app/assets/javascripts/surveys/modules/Survey.js

```diff
@@ -552,12 +556,12 @@ const Survey = {
     });
   },
 
-  addConditionalQuestionToStore(questionId, $question) {
-    if (typeof this.surveyConditionals[questionId] !== 'undefined') {
-      this.surveyConditionals[questionId].children.push($question[0]);
+  addConditionalQuestionToStore(question_id, $question) {
```

Comment:
These ones were actually already camelCase, so you need to change them back to that.

This PR will be ready when `question_id` doesn't show up at all in the changes.